### PR TITLE
Activate repositories callbacks

### PR DESCRIPTION
**Pull request by renanivo**
The callback view just logs what it receives.

### CODE REVIEW

File: houserules/repositories/views.py [hunk 1]
```diff
@@ -1,17 +1,45 @@
...
+                'repository_list': (
+                    {
+                        'instance': repository,
+                        'form': (
+                            RepositoryForm(instance=repository)
+                            if repository.permissions.first().is_admin
+                            else ReadOnlyRepositoryForm(instance=repository)
+                        )
+                    }
+                    for repository in repositories
```
Comment (drgarcia1986) [inline houserules/repositories/views.py]: What do you think to run this comprehension before `render` function ?

Comment (renanivo) [inline houserules/repositories/views.py]: If that improves readability, 👍 .

**Comment by renanivo** [comment]
updated

**Comment by drgarcia1986** [comment]
👍

**Pull request merged**